src/cljs/witan/ui/components/login.cljs

```diff
@@ -95,16 +97,11 @@
     [:span {:id "reset-instructions"} (get-string :string/forgotten-instruction)]]
    [:form {:class "pure-form"
            :on-submit (fn [e]
-                        (comment (set! (.-innerText (. js/document (getElementById "reset-instructions"))) (get-string :string/reset-submitted))
-                                 (set! (.-innerText (. js/document (getElementById "reset-button"))) (get-string :string/thanks))
-                                 (set! (.-disabled (. js/document (getElementById "reset-button"))) true)
-                                 (set! (.-disabled (. js/document (getElementById "reset-input"))) true))
-                        (.open js/window
-                               (str
-                                "mailto:witan@mastodonc.com?subject=[Witan Password Reset Request]"
-                                "&body=Please reset the password for the following email address: "
-                                (.-value (.getElementById js/document "reset-input"))) "resetEmailWindow" "height=400,width=600,left=10,top=10")
-                        #_(venue/raise! owner :event/reset-password (.-value (om/get-node owner "reset-email")))
+                        (set! (.-innerText (. js/document (getElementById "reset-instructions"))) (get-string :string/reset-submitted))
+                        (set! (.-innerText (. js/document (getElementById "reset-button"))) (get-string :string/thanks))
+                        (set! (.-disabled (. js/document (getElementById "reset-button"))) true)
+                        (set! (.-disabled (. js/document (getElementById "reset-input"))) true)
+                        (controller/raise! :user/reset-password (.-value (. js/document (getElementById "reset-input"))))
```

Comment:
This code is legacy -- I wouldn't toggle properties like this anymore. It's harmless just non-idiomatic. Will fix it one day :angel: